README.md

```diff
@@ -127,7 +127,7 @@ Which produces:
   "environment" : "undefined"
 }
 {
-  "timestamp" : "2017-02-02T12:32:38.252+00:00",
+  "timestamp" : "2017-02-02T12:32:38.2.2.10:00",
```

Comment:
What's happening here?

Reply:
Dear me! Bogus shell script for version update. Will include a fix in this PR as it's a super minor